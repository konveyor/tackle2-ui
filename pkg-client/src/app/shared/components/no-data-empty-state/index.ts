export { NoDataEmptyState } from "./no-data-empty-state";
