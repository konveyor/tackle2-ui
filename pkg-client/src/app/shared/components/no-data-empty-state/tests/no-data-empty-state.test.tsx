import React from "react";
import { shallow } from "enzyme";
import { NoDataEmptyState } from "../no-data-empty-state";

describe("NoDataEmptyState", () => {
  it("Renders without crashing", () => {
    const wrapper = shallow(
      <NoDataEmptyState title="my title" description="my description" />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
