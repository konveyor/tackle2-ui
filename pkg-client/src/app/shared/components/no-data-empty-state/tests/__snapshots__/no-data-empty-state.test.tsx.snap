// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoDataEmptyState Renders without crashing 1`] = `
<EmptyState
  variant="small"
>
  <EmptyStateIcon
    icon="test-file-stub"
  />
  <Title
    headingLevel="h2"
    size="lg"
  >
    my title
  </Title>
  <EmptyStateBody>
    my description
  </EmptyStateBody>
</EmptyState>
`;
